infracost-usage-example.yml

```diff
@@ -719,6 +719,9 @@ resource_usage:
   google_pubsub_topic.my_topic:
     monthly_message_data_tb: 7.416 # Monthly amount of message data published to the topic in TB.
 
+  google_redis_cluster.my_redis_cluster:
+    backup_storage_gb: 1000 # Amount of backup storage in GB.
```

Comment:
I forgot to mention we should add usage examples to these files. I've added them not.